src/pkg/count/count.go

```diff
@@ -138,3 +108,31 @@ func (b *Bus) TotalValues() map[string]int64 {
 
 	return b.Values()
 }
+
+// ---------------------------------------------------------------------------
+// compliance with callbacks and external packages
+// ---------------------------------------------------------------------------
+
+// AdderFor returns a func that adds any value of i
+// to the bus using the given key.
+func (b *Bus) AdderFor(k Key) func(i int64) {
+	return func(i int64) {
+		b.Add(k, i)
+	}
+}
+
+type plainAdder struct {
+	bus *Bus
+}
+
+func (pa plainAdder) Add(k string, n int64) {
+	if pa.bus == nil {
+		return
+	}
+
+	pa.bus.Add(Key(k), n)
+}
+
+func (b *Bus) PlainAdder() *plainAdder {
```

Comment:
Could you add a short desc for this? Right now it sounds like a snake species! 🐍

Reply:
hah!  Sure, can do.